### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by dannymout**
Changed `Moya.Error` in [Basic](https://github.com/Moya/Moya/blob/master/docs/Examples/Basic.md) to `MoyaError`. Changed `verbose` in [Plugins](https://github.com/Moya/Moya/blob/master/docs/Plugins.md#built-in-plugins) to `isVerbose`.

### CODE REVIEW

**Comment by scottrhoyt** [comment]
Thanks @dannymout !

File: docs/Plugins.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Moya ships with some default plugins which can be used for common functions: aut
 You can use a plugin simply declaring it during the initialization of the provider:
 
 ```swift
-let provider = MoyaProvider<GitHub>(plugins: [NetworkLoggerPlugin(verbose: true)])
+let provider = MoyaProvider<GitHub>(plugins: [NetworkLoggerPlugin(isVerbose: true)])
```
Comment (BasThomas) [inline docs/Plugins.md]: The initializer still uses `verbose` as its parameter, so this is incorrect.

Comment (BasThomas) [inline docs/Plugins.md]: This was discussed in #841

Comment (BasThomas) [inline docs/Plugins.md]: Created #990 to change it back :)

Comment (dannymout) [inline docs/Plugins.md]: Sorry, didn't know that. @BasThomas

Comment (BasThomas) [inline docs/Plugins.md]: No problem. :)

**Pull request merged**